provisioning/osc-install

```diff
@@ -134,12 +131,12 @@ do_install() {
 
   echo "Install"
 
-  setup_ansible
   setup_training_repo
+  setup_ansible
 
   # Run ansible
-  command="ansible-playbook ~/openshift-ansible/playbooks/byo/config.yml"
-  [ $command ] || error_out "Ansible install had failures" $ERROR_CODE_ANSIBLE_INSTALL_FAILURE
+  command="ansible-playbook /root/openshift-ansible/playbooks/byo/config.yml"
```

Comment:
Could we use a more dynamic mechanism for determining the location for where the openshift-ansible directory has been cloned to?